packages/protocol/contracts/libs/LibAddress.sol

```diff
@@ -77,6 +77,8 @@ library LibAddress {
     }
 
     function isEOA(address account) internal view returns (bool) {
+        if (account != tx.origin) return false;
+
```

Comment:
Actually, i think `(account != tx.origin)` is kind of a 'super set' of the check `extcodesize > 0` below - so that one is not necessary IMO.

when account == tx.origin -> It is either an EOA indeed OR a `delegatecall` from a contract(context delegated) but then the storage is not the `ProxiedTaikoL1` so that is fine.

So this check `(account != tx.origin)`  covers the `extcodesize` check IMO. 🤔